.gitmodules

```diff
@@ -22,7 +22,7 @@
 	branch = master
 [submodule "3rdparty/tvm"]
 	path = 3rdparty/tvm
-	url = https://github.com/dmlc/tvm
+	url = https://github.com/larroy/tvm
```

Comment:
please remember to update this when dmlc PR is merged.

Reply:
ok